core-bundle/tests/Contao/CombinerTest.php

```diff
@@ -143,9 +140,9 @@ public function testHandlesSpecialCharactersWhileFixingTheFilePaths(): void
             EOF;
 
         $expected = <<<'EOF'
-            test1 { background: url("../../\"test\"/foo.bar") }
-            test2 { background: url("../../\"test\"/foo.bar") }
-            test3 { background: url('../../"test"/foo.bar') }
+            test1 { background: url("../../public/\"test\"/foo.bar") }
+            test2 { background: url("../../public/\"test\"/foo.bar") }
+            test3 { background: url('../../public/"test"/foo.bar') }
```

Comment:
I’d change the `['name' => 'public/"test"/file.css']` to `['name' => '"test"/file.css']` a few lines below and revert these changes here to make the intended behavior more clear.
But this is just a nitpick, feel free to ignore.